defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+# ssh connection settings
+molecule_openstack_ci_ssh_user: cloud-user
+molecule_openstack_ci_ssh_port: 22
+
+# run_hash will be appended to security group name
+molecule_openstack_ci_security_group_name: molecule
+molecule_openstack_ci_security_group_description:
+  "Security group for testing Molecule"
```

Comment:
For "testing Molecule" for for "Molecule testing" or for "testing with Molecule"?